regtests/ww3_ts1/input_nl5_gaussian/switch

```diff
@@ -0,0 +1 @@
+F90 NOGRB DIST MPI PR0 FLX0 LN0 ST0 NL5 BT0 DB0 MLIM TR0 BS0 IC0 IS0 REF0 XX0 WNT1 WNX1 RWND CRT1 CRX1 O0 O1 O2 O4 O5 O6 O7
```

Comment:
Should this be switch_MPI instead of just switch?

Reply:
Also, since this is just a single point test, I don't really think you can do much with MPI/DIST here, so perhaps the DIST MPI  should just be SHRD and then maybe add a switch_OMP as maybe OMP would help speed things up?

Reply:
yes, for these single point tests, it is not necessary to use MPI/DIST. I revised the switch file by replacing 'MPI DIST' with 'SHRD'.

Reply:
I still see MPI DIST not SHRD here?